atlas/map.go

```diff
@@ -207,11 +204,12 @@ func (m Map) Encode(ctx context.Context, tile *slippy.Tile) ([]byte, error) {
 					}
 				}
 
+				//TODO: makevalid issues for 4326
 				// check if we need to clip and if we do build the clip region (tile extent)
 				var clipRegion *geom.Extent
```

Comment:
I had to comment out these lines in order to get rendering looking appropriate (L209-231)